static_src/main.js

```diff
@@ -31,10 +31,18 @@ var updateModel = function() {
                 }
 
                 if (!_.findByKey(category.children, c[1])) {
-                    category.children.push({
-                        key: c[1],
-                        active: true,
-                    });
+                    if (state == null) {
```

Comment:
As a general rule, you should never use `==` in JavaScript (because it is easily confused with `===`). Either use `state === null` or just `state`. In this case the latter is probably the better choice.